pulp_file/tests/functional/api/test_crud_remotes.py

```diff
@@ -1,204 +1,105 @@
-
-        Get the current remote policy.
-        Attempt to update the remote policy to an invalid value.
-        Verify the policy remains the same.
-        """
-        remote = self.remote_api.read(self.remote["pulp_href"]).to_dict()
-        with self.assertRaises(ApiException):
-            self.remote_api.partial_update(self.remote["pulp_href"], {"policy": utils.uuid4()})
-        self.remote.update(self.remote_api.read(self.remote["pulp_href"]).to_dict())
-        self.assertEqual(remote["policy"], self.remote["policy"], self.remote)
-
-
-def _gen_verbose_remote():
-    """Return a semi-random dict for use in defining a remote.
-
-    For most tests, it"s desirable to create remotes with as few attributes
-    as possible, so that the tests can specifically target and attempt to break
-    specific features. This module specifically targets remotes, so it makes
-    sense to provide as many attributes as possible.
-
-    Note that 'username' and 'password' are write-only attributes.
-    """
-    attrs = gen_file_remote(url=choice((FILE_FIXTURE_MANIFEST_URL, FILE2_FIXTURE_MANIFEST_URL)))
-    attrs.update(
-        {
-            "password": utils.uuid4(),
-            "username": utils.uuid4(),
-            "policy": choice(ON_DEMAND_DOWNLOAD_POLICIES),
-        }
+@pytest.mark.parallel
+def test_remote_crud_workflow(file_remote_api_client):
+    remote_data = {"name": str(uuid.uuid4()), "url": "http://example.com"}
+    remote = file_remote_api_client.create(remote_data)
+    assert remote.url == remote_data["url"]
+    assert remote.name == remote_data["name"]
+
+    with pytest.raises(ApiException) as exc:
+        file_remote_api_client.create(remote_data)
+    assert exc.value.status == 400
+    assert json.loads(exc.value.body) == {"name": ["This field must be unique."]}
+
+    update_response = file_remote_api_client.partial_update(
+        remote.pulp_href, {"url": "https://example.com"}
+    )
+    task = monitor_task(update_response.task)
+    assert task.created_resources == []
+
+    remote = file_remote_api_client.read(remote.pulp_href)
+    assert remote.url == "https://example.com"
+
+    all_new_remote_data = {"name": str(uuid.uuid4()), "url": "http://example.com"}
+    update_response = file_remote_api_client.update(remote.pulp_href, all_new_remote_data)
+    task = monitor_task(update_response.task)
+    assert task.created_resources == []
+
+    remote = file_remote_api_client.read(remote.pulp_href)
+    assert remote.name == all_new_remote_data["name"]
+    assert remote.url == all_new_remote_data["url"]
+
+    delete_reponse = file_remote_api_client.delete(remote.pulp_href)
+    monitor_task(delete_reponse.task)
+
+    with pytest.raises(ApiException) as exc:
+        file_remote_api_client.read(remote.pulp_href)
+    assert exc.value.status == 404
+
+
+@pytest.mark.parallel
+def test_negative_create_file_remote_with_invalid_parameter(file_remote_api_client):
```

Comment:
```suggestion
def test_create_file_remote_with_invalid_parameter(file_remote_api_client):
```
I do not see the value in that piece of the name. The fact that it fails is well encoded (and readable) in the assertions.